contracts/curve/types/JumpRateUtilizationCurve6.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.13;
+
+import "../CurveMath6.sol";
+import "../../number/types/Fixed6.sol";
+
+/// @dev JumpRateUtilizationCurve6 type
+struct JumpRateUtilizationCurve6 {
+    Fixed6 minRate;
+    Fixed6 maxRate;
+    Fixed6 targetRate;
+    UFixed6 targetUtilization;
+}
+using JumpRateUtilizationCurve6Lib for JumpRateUtilizationCurve6 global;
+type JumpRateUtilizationCurve6Storage is bytes32;
+using JumpRateUtilizationCurve6StorageLib for JumpRateUtilizationCurve6Storage global;
+
+/**
+ * @title JumpRateUtilizationCurve6Lib
+ * @notice Library for the base-6 Jump Rate utilization curve type
+ */
+library JumpRateUtilizationCurve6Lib {
+    /**
+     * @notice Computes the corresponding rate for a utilization ratio
+     * @param utilization The utilization ratio
+     * @return The corresponding rate
+     */
+    function compute(JumpRateUtilizationCurve6 memory self, UFixed6 utilization) internal pure returns (Fixed6) {
+        UFixed6 targetUtilization = self.targetUtilization;
```

Comment:
When we're reading directly from an already in-memory struct we don't need to cache the variable here -- so feel free to use `self.targetUtilization` directly below.